CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## unreleased
 
+* Updated all dependencies to the latest.
+* Updated node requirement to `^14.15.0 || ^16.10.0 || >=18.0.0`
```

Comment:
```suggestion
* Updated the minimum version of Node to `14.15.0`.
```

Reply:
I knew that some module drops node 17(babel-jest, eslint-plugin-jest, etc.). 
You can check the detailed list in WRP-11567.

Is it OK to describe the minimum version only?